test/verify/check-machines

```diff
@@ -28,6 +28,9 @@ from testlib import *
 from netlib import NetworkHelpers
 from storagelib import StorageHelpers
 from machinesxmls import *
+from datetime import datetime
+
+distrosWithoutSnapshot = ["centos-8-stream", "debian-stable", "fedora-31", "fedora-32", "fedora-coreos", "fedora-testing", "rhel-7-9", "rhel-8-3", "rhel-8-3-distropkg", "ubuntu-2004", "ubuntu-stable"]
```

Comment:
How sad :crying_cat_face: You can drop "rhel-7-9", we don't support rhel 7 from master any more.